OSDOCS#4348: Adding release note for IBM Cloud restricted installs (#70636)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

For more information, see xref:../installing/installing_ibm_cloud_public/user-managed-encryption-ibm-cloud.adoc#user-managed-encryption-ibm-cloud[User-managed encryption for IBM Cloud].

[id="ocp-4-15-installation-and-update-ibm-cloud-restricted-install"]
==== Installing a cluster on IBM Cloud with limited internet access

> You can now install a cluster on {ibm-cloud-name} when the environment has limited access to to the internet, as in the case of a disconnected or restricted network cluster. With this type of installation, you create a registry that mirrors the contents of the {product-title} image registry and contains the installation media. You can create this registry on a mirror host, which can access both the internet and your restricted network.

Reviewer: ```suggestion
You can now install a cluster on {ibm-cloud-name} when the environment has limited access to the internet, as in the case of a disconnected or restricted network cluster. With this type of installation, you create a registry that mirrors the contents of the OpenShift Container Platform installation images. You can create this registry on a mirror host, which can access both the internet and your restricted network.
```

Author: Thank you. Please take one more look.

Author: Confirmed with Steven in Slack that latest updates are good.

For more information, see xref:../installing/installing_ibm_cloud_public/user-managed-encryption-ibm-cloud.adoc#user-managed-encryption-ibm-cloud[User-managed encryption for IBM Cloud].

[id="ocp-4-15-installation-and-update-ibm-cloud-restricted-install"]
==== Installing a cluster on IBM Cloud with limited internet access

> You can now install a cluster on {ibm-cloud-name} when the environment has limited access to the internet, as in the case of a disconnected or restricted network cluster. With this type of installation, you create a registry that mirrors the contents of the {product-title} installation images. You can create this registry on a mirror host, which can access both the internet and your restricted network.

Reviewer 2: Just playing around with the wording of the first sentence. No real reason to change if you don't want to:

```suggestion
You can now install a cluster on {ibm-cloud-name} in environments with limited internet access, such as disconnected or restricted network clusters. With this type of installation, you create a registry that mirrors the contents of the {product-title} installation images. You can create this registry on a mirror host, which can access both the internet and your restricted network.
```

Author: Thanks @stevsmit I think I will revise slightly after looking at it again, but not take it verbatim.

Author: Confirmed with Steven in Slack that the latest changes are good.
